### PR TITLE
SPCTR-2481 Fix: Modal - Info box as a innerBlock not loading in modal block when gutenberg is activated.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description
Fixed bug for info-box not loading as a innerBlock in Modal block when gutenberg is activated.

### Screenshots
https://d.pr/v/0HaOvO

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,6 +200,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 * Fix: Global - Excluded BR Tags from buttons' text.
 * Fix: Global - Cartflows templates throw an error in the editor's Spectra Page Settings.
 * Fix: Global - Fonts was not loading on 404 page when Spectra blocks used in Astra theme's header/footer.
+* Fix: Modal - Info box as a innerBlock not loading in modal block when gutenberg is activated
```
Comment (RajkiranBagal) [inline README.md:203]:
```suggestion
* Fix: Modal - Info box as a inner block not loading in the Modal block when Gutenberg is activated.
```

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md:209]: Added

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Comment by aapatre** [comment]
Have you confirmed backward-compatibility?

**Comment by rajmohanreddy774** [comment]
> Have you confirmed backward-compatibility?

Yes backward-compatibility check.

**Review by aapatre — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

File: src/blocks/modal/render.js [hunk 2]
```diff
@@ -96,20 +84,15 @@ const Render = ( props ) => {
...
 		<div className='uagb-modal-trigger'>
-			{ '' !== icon && (
-				renderSVG( icon, setAttributes )
-			) }
+			{'' !== icon && renderSVG( icon, setAttributes ) }
 		</div>
 	);
 
-	const defaultedAlt = ( iconImage && iconImage?.alt ) ? iconImage?.alt : '';
+	const defaultedAlt = ( iconImage && iconImage )?.alt ? iconImage?.alt : '';
```
Comment (rahulvarma722) [inline src/blocks/modal/render.js:91]:
```suggestion
	const defaultedAlt = iconImage?.alt ? iconImage.alt : '';
```
Please check this once

File: src/blocks/modal/modalConfig.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import { __ } from '@wordpress/i18n';
+
+export const defaultContent = [
```
Comment (rahulvarma722) [inline src/blocks/modal/modalConfig.js:3]: Add proper description for this variable like when you add first time this content will show

File: src/blocks/modal/modalConfig.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            paddingBtnLeft: 28,
+            btnBorderTopLeftRadius: 3,
+            btnBorderTopRightRadius: 3,
+            btnBorderBottomLeftRadius: 3,
+            btnBorderBottomRightRadius: 3,
+        },
+    ],
+];
+
+export const excludeBlocks = [
```
Comment (rahulvarma722) [inline src/blocks/modal/modalConfig.js:36]: Add proper description for this

File: src/blocks/modal/render.js [hunk 5]
```diff
@@ -96,20 +84,15 @@ const Render = ( props ) => {
...
 	let imageIconHtml = '';
 	let url = '';
-	useEffect( () => {
-		getImageHeightWidth( url, setAttributes )
-	},[ imageSize ] )
-	if ( iconImage && iconImage.url ) {
 
+	if ( iconImage && iconImage.url ) {
```
Comment (rahulvarma722) [inline src/blocks/modal/render.js:95]:
```suggestion
	if ( iconImage?.url ) {
```

File: src/blocks/modal/modalConfig.js (hunk 3 above)
Comment (rajmohanreddy774) [reply src/blocks/modal/modalConfig.js:4]: Added

File: src/blocks/modal/modalConfig.js (hunk 4 above)
Comment (rajmohanreddy774) [reply src/blocks/modal/modalConfig.js:38]: Added

**Review by rahulvarma722 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**